python/requirements.in

```diff
@@ -2,62 +2,62 @@ black
 beautifulsoup4>=4.9.3
 boto3>=1.28.1
 boto3-stubs[s3]>=1.28.1
-cached-property==1.5.2
-contextily==1.3.0
+cached-property
+contextily
 dbt-core
 dbt-postgres
 diagrams>=0.23.3
-faker==23.1.0
-folium==0.14.0
+faker
+folium
 GDAL==3.8.3
 geopandas
-geoalchemy2==0.14.1
+geoalchemy2
```

Comment:
curious, what's this package used for?

Reply:
it _might_ be used in the QA app

Reply:
like `tabulate`, I suspect it's sort of an "optional" dependency of another package and isn't needed until you try to use certain functions